### DISCUSSION TITLE
Why was the Nurse Rostering example deleted?

### DISCUSSION THREAD

**Discussion by slxnxl** [TimefoldAI/timefold-solver#812] [General]
Why did you delete it https://github.com/TimefoldAI/timefold-solver/pull/799 ? 
Will there be examples with Consecutive collectors?

**Comment by triceo** [comment]
Hello @slxnxl, and thank you for your question.

We've converted the examples to [Quickstarts](https://github.com/TimefoldAI/timefold-quickstarts/), modernizing them in the process. Not only can you find examples with consecutive collectors there, we have also taken the one remaining experimental collector ("consecutive intervals", now called "connected ranges") and made an official collector out of it.

Some of that work may not yet be visible in Timefold Solver 1.9.0, but it will all be released in Timefold Solver 1.10.0 next month.

**Reply by kentbill** [reply]
I think there is an use case - the **Project Job Scheduling** example where a job has Predecessor(s)/Successor(s), and it seems that there is no similar example to refer to in Quickstarts.

**Reply by triceo** [reply]
That is exactly why I'm saying you should wait for 1.10 - it will be there. (But for the record - we didn't migrate _every_ example. Some were either too simple, or duplicated what we already had in the quickstarts. So even though no functionality was lost, it may not be in the same place or with the same name as where it was before.)

**Comment by slxnxl** [comment]
Thank you for your reply! I will be waiting for updates!
